app/Nova/Resources/UiTiDv1.php

```diff
@@ -90,4 +95,30 @@ private function getActionUrlTemplates(): array
             )
         );
     }
+
+    public function actions(NovaRequest $request): array
+    {
+        return [
+            (new ActivateUitIdv1Client())
```

Comment:
I would also add a confirmation text here (same remark for the Auth0 activate action)